### DISCUSSION TITLE
Build

### DISCUSSION THREAD

**Discussion by seloner** [expo/expo#17384] [Archived: Questions]
How can I build the expo app and I have the apk instant locally and not upload to expo servers . I dont want to use EAS because I need to wait 30 minutes for every build .

**Comment by Robert27** [comment]
Hello,

You can also run the EAS build locally, more details here:
https://docs.expo.dev/build-reference/local-builds/

Basically, you need one of these commands:
```bash
eas build --platform android --local
eas build --platform ios --local
```

I hope this helps you :)
